.eslintignore

```diff
@@ -1,11 +1,10 @@
-localforage.js
+localstorage.js
 node_modules
 .cache-loader
 coverage
 /pipelines/
 /trustedPipelines/
 /themes/*/extensions/
-/extensions/*
```

Comment:
Why did you remove this line? Extensions other than the three defined below SHOULD be gitignored! Please bring that line back.

Reply:
no. its about eslint ignoring, which we dont want to ignore, since eslint does not work for attached extensions